src/redux/answers/answers.actions.js

```diff
@@ -9,13 +9,15 @@ import axios from 'axios';
 import {setAlert} from '../alert/alert.actions';
 import config from "../../config";
 
+import censorBadWords from '../../services/censorBadWords'
+
 export const getAnswers = (id) => async (dispatch) => {
   try {
     const res = await axios.get(config.BASE_URL + `/api/posts/answers/${id}`);
 
     dispatch({
       type: GET_ANSWERS,
-      payload: res.data.data,
+      payload: res.data.data.map(({body, ...rest})=>({...censorBadWords({body}), ...rest})),
```

Comment:
Here we are looping on the complete response and then changing the text, and then while rendering we are anyways looping through the result.

The better approach would be to convert the text while rending so that we can avoid an extra loop, and avoid a delete and an insert operation